api_bonus.yaml

```diff
@@ -71,39 +45,9 @@ paths:
       operationId: startBonusActivationProcedure
       summary: Start bonus activation request procedure
       responses:
-        "201":
-          description: Request created.
-          schema:
-            $ref: "#/definitions/InstanceId"
-          headers:
-            Location:
-              type: string
-              description: |-
-                Location (URL) of created request resource.
-                A GET request to this URL returns the request status and details.
-        "202":
-          description: Processing request.
-          schema:
-            $ref: "#/definitions/InstanceId"
-        "401":
```

Comment:
this API could return 401 with `session_token` evaluation (`bearerSessionTokenAuth`) or 500 if something goes wrong with redis client